submissions/asaMitaka/FriendsApp/script.js

```diff
@@ -0,0 +1,87 @@
+const url = 'https://randomuser.me/api/?results=12';
+const content = document.querySelector('.articleBlock');
+const form = document.forms;
```

Comment:
Why do you have 2 forms? You should use 1 form and get it by id or something, don't rely on `document.forms`.